app/src/main/java/com/framgia/soundclound/screen/detailalbum/DetailAlbumActivity.java

```diff
@@ -15,6 +15,9 @@
  */
 
 public class DetailAlbumActivity extends AppCompatActivity {
+    public static final int REQUEST_UPDATE_DATA = 10001;
+    public DetailAlbumModelView mDetailAlbumModelView;
```

Comment:
WARNING: Variable 'mDetailAlbumModelView' must be private and have accessor methods.